input/resources/structuredefinition-encounter-description.xml

```diff
@@ -21,7 +21,7 @@
   <differential>
```

Comment:
Change the resource status - it is no longer active when deprecated.

Reply:
The only available status codes are Active, Draft, Retired and Unknown, see https://hl7.org/fhir/R4/structuredefinition.html. Deprecated in my view lies somewhere between Active and Retired... what's the right status?

Reply:
discussed with Dusi. Changed status to retired.